src/server.cpp

```diff
@@ -1639,13 +1638,13 @@ bool CServer::CreateLevelsForAllConChannels ( const int                        i
         for ( int j = 0; j < iNumClients; j++ )
         {
             // update and get signal level for meter in dB for each channel
-            const double dCurSigLevelForMeterdB = vecChannels[vecChanIDsCurConChan[j]].
+            const float dCurSigLevelForMeterdB = vecChannels[vecChanIDsCurConChan[j]].
```

Comment:
Have you checked that the clipping indicator still works fine with your changes?

Reply:
No.

Reply:
We have to test this functionality before I can merge your pull request.

Reply:
Tested that clipping indicator works for both Input and Channel strips.

Reply:
Test OK (MacOSX)